scripts/contentScript.js

```diff
@@ -380,6 +380,99 @@ async function FilesInPrBitbucket(response) {
 	});
 }
 
+const githubHunkHighlight = async (apiResponses) => {
+
+	// Todo : optomization needed to not highlight next deleted line space if not present in response.
+	let getFileName = Array.from(document.querySelectorAll('div[data-tagsearch-path]'));
+	getFileName.forEach(async (item) => {
+		let FileContent = item.getAttribute('data-tagsearch-path');
+		if (FileContent) {
+
+			let matchEncrypted = await sha256(FileContent);
+			let foundFile = null;
+			for (k in apiResponses["hunkinfo"]) {
+				let item = apiResponses["hunkinfo"][k];
+				if (item.file === matchEncrypted) {
+					foundFile = item;
+				}
+			}
+			if (foundFile) {
+				let value = Array.from(item.getElementsByTagName('tr'));
+				let changeBg = false;
+
+				value.forEach((items) => {
+					let secondRow = Array.from(items.getElementsByTagName('td'));
+					secondRow.forEach((item) => {
+						let buttonId = item.querySelector('button[data-line]');
+						if (buttonId) {
+							let dataLineValue = buttonId.getAttribute('data-line');
+							let tableContent = items.querySelector("td[data-split-side='left']");
```

Comment:
Please use `const` wherever possible. It is not a good practice to use `let`. I don't think any of these variables are being reassigned (except `foundFile`). Using `let` just makes the code less readable.